Regression/WarpX-tests.ini

```diff
@@ -427,14 +430,35 @@ compileTest = 0
 doVis = 0
 compareParticles = 1
 particleTypes = electrons positrons
+outputFile = diags/d00040
 analysisRoutine = Examples/Tests/Langmuir/analysis_langmuir_multi.py
 analysisOutputImage = langmuir_multi_analysis.png
 tolerance = 5.e-11
 
+[Langmuir_multi_psatd_cc]
```

Comment:
`cc` is a bit ambiguous, in particular it is used in WarpX for Cell-Centered. Could you replace `cc` with e.g., `CurrentCorrection`?